assets/test/specs/form.spec.js

```diff
@@ -24,6 +29,9 @@ var setup = function () {
   $textInputExample = $('#text-input-example');
   $radioYesElement = $('#radio-yes');
   $radioNoElement = $('#radio-no');
+  $radioGroupThis = $('#radio-this');
+  $radioGroupThat = $('#radio-that');
```

Comment:
These are token values - is the question should they be `$radioGroupThisElement` or that they be like those already used, i.e. `$radioGroupNoElement`?